packages/components/index.d.ts

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { ReactNode } from 'react';
+
+export interface ViewportTrackerProps {
+    children: React.ReactNode
+}
+export class ViewportTracker extends React.Component<ViewportTrackerProps> {}
+
+
+export interface ViewportAwareComponentProps {
+    /**
+     * Determines pre-triggering of inViewport. Useful for rendering components beforehand to improve user experience. A ratio of 0.5 means that the effective viewport will be twice the size of the real viewport.
+     */
+    preTriggerRatio?: number = 0
+    /**
+     * Invoked when the component enters the viewport.
+     */
+    onViewportEnter?: () => void
+    /**
+     * Invoked when the component leaves the viewport.
+     */
+    onViewportLeave?: () => void
+    /**
+     * Allows access to the reference of the wrapped component.
+     */
+    innerRef?: React.Ref
+}
+export class ViewportAware<P> extends React.Component<ViewportAwareComponentProps & P> {}
```

Comment:
Why are all of the classes and interfaces exported. Isn't the `const Viewport` export at the bottom sufficient?